core/modules/editor/factory.js

```diff
@@ -162,6 +162,9 @@ function editTextWidgetFactory(toolbarEngine,nonToolbarEngine) {
 	Compute the internal state of the widget
 	*/
 	EditTextWidget.prototype.execute = function() {
+		this.editorStateQualifier = this.getVariable("storyTiddler");
+		$tw.editorManager = $tw.editorManager || [],
+		$tw.editorManager[this.editorStateQualifier] = $tw.editorManager[this.editorStateQualifier] || {};
```

Comment:
The above 2 lines don't make much sense to me. You copy the `$tw.editorManager[this.editorStateQualifier]` variable to itself, or init it with an empty object. 

`$tw` is a global structure. So initialisation should be done at startup or registering elements. ... But imo not as a "side-effect" in execute.

Reply:
aha, I was initializing it differently originally, like "if it's undefined, then it's `[]` "

I think I could move it to a startup module - or - like the keyboardManager - a "global" module-type. I haven't looked at what that global type is, but I imagine the editorManager to work similarily